src/victory-util/collection.js

```diff
@@ -66,5 +68,28 @@ export default {
     return segments.filter((segment) => {
       return Array.isArray(segment) && segment.length > 0;
     });
+  },
+
+  /**
+   * Takes an array of arrays. Returns whether each subarray has equivalent items.
+   * Each subarray should have two items. Used for componentShouldUpdate functions.
+   *
+   * Example:
+   * const propComparisons = [
+   *   [x, nextProps.x],
+   *   [y, nextProps.y],
+   *   [style, this.style]
+   * ];
+   *
+   * allSetsEqual(propComparisons);
+   * => true
+   *
+   * @param {Array}    itemSets     An array of item sets
+   * @returns {Boolean}             Whether all item comparisons are equal
+   */
+  allSetsEqual(itemSets) {
```

Comment:
Do you know whether it's more performant to use `===` when you're sure it will be a simple equality check? How much overhead is  `_.isEqual` adding in these cases?